src/components/scenes/SendConfirmationScene.js

```diff
@@ -389,59 +391,36 @@ export class SendConfirmation extends Component<Props, State> {
-      const cryptoFeeDisplayDenomAmount = bns.mul(cryptoFeeExchangeDenomAmount, exchangeToDisplayMultiplierRatio)
-      const cryptoFeeString = `${cryptoFeeSymbol} ${cryptoFeeDisplayDenomAmount}`
-      const fiatFeeSymbol = secondaryInfo.displayDenomination.symbol ? secondaryInfo.displayDenomination.symbol : ''
-      const exchangeConvertor = convertNativeToExchange(this.props.parentExchangeDenomination.multiplier)
-      const cryptoFeeExchangeAmount = exchangeConvertor(parentNetworkFee)
-      const fiatFeeAmount = convertCurrencyFromExchangeRates(
-        exchangeRates,
-        this.props.parentExchangeDenomination.name,
-        secondaryInfo.exchangeCurrencyCode,
-        parseFloat(cryptoFeeExchangeAmount)
-      )
-      const fiatFeeAmountString = fiatFeeAmount.toFixed(2)
-      const fiatFeeAmountPretty = bns.toFixed(fiatFeeAmountString, 2, 2)
-      const fiatFeeString = `${fiatFeeSymbol} ${fiatFeeAmountPretty}`
-      return sprintf(s.strings.send_confirmation_fee_line, cryptoFeeString, fiatFeeString)
-    }
-
-    if (networkFee && bns.gt(networkFee, '0')) {
-      const cryptoFeeSymbol = primaryInfo.displayDenomination.symbol ? primaryInfo.displayDenomination.symbol : ''
-      // multiplier for display denomination
-      const displayDenomMultiplier = primaryInfo.displayDenomination.multiplier
-      // multiplier for EXCHANGE denomination
-      const cryptoFeeMultiplier = this.props.primaryExchangeDenomination.multiplier
-      // fee amount in exchange denomination
-      const cryptoFeeExchangeDenomAmount = networkFee ? convertNativeToDisplay(cryptoFeeMultiplier)(networkFee) : ''
-      const exchangeToDisplayMultiplierRatio = bns.div(cryptoFeeMultiplier, displayDenomMultiplier, DIVIDE_PRECISION)
-      const cryptoFeeDisplayDenomAmount = bns.mul(cryptoFeeExchangeDenomAmount, exchangeToDisplayMultiplierRatio)
-      const cryptoFeeString = `${cryptoFeeSymbol} ${cryptoFeeDisplayDenomAmount}`
-      const fiatFeeSymbol = secondaryInfo.displayDenomination.symbol ? secondaryInfo.displayDenomination.symbol : ''
-      const exchangeConvertor = convertNativeToExchange(primaryInfo.exchangeDenomination.multiplier)
-      // amount in EXCHANGE denomination
-      const cryptoFeeExchangeAmount = exchangeConvertor(networkFee)
-      const fiatFeeAmount = convertCurrencyFromExchangeRates(
-        exchangeRates,
-        this.props.currencyCode,
-        secondaryInfo.exchangeCurrencyCode,
-        parseFloat(cryptoFeeExchangeAmount)
-      )
-      const fiatFeeAmountString = fiatFeeAmount.toFixed(2)
-      const fiatFeeAmountPretty = bns.toFixed(fiatFeeAmountString, 2, 2)
-      const fiatFeeString = `${fiatFeeSymbol} ${fiatFeeAmountPretty}`
-      return sprintf(s.strings.send_confirmation_fee_line, cryptoFeeString, fiatFeeString)
+      denomination = parentDisplayDenomination
+      exchangeDenomination = this.props.parentExchangeDenomination
+      usedNetworkFee = parentNetworkFee
+      currencyCode = this.props.currencyCode
```

Comment:
I think you got this line backwards with the `else` statement. This one should be `this.props.parentExchangeDenomination.name`?